service/host.go

```diff
@@ -20,11 +20,20 @@ var (
 		evergreen.HostRunning,
 		evergreen.HostQuarantined,
 		evergreen.HostDecommissioned,
+		evergreen.HostTerminated,
 	}
 )
 
 const (
-	IncludeSpawnedHosts = "includeSpawnedHosts"
+	IncludeSpawnedHosts            = "includeSpawnedHosts"
```

Comment:
This looks to be unused

Reply:
Line 76 uses it, but it looks like that's the only place. Should I hardcode the string there instead?

Reply:
Oops disregard this, I did not search correctly